gazebo_ros/test/test_conversions.cpp

```diff
@@ -15,6 +15,7 @@
 #include <gazebo_ros/conversions/builtin_interfaces.hpp>
 #include <gazebo_ros/conversions/sensor_msgs.hpp>
 #include <gazebo_ros/conversions/geometry_msgs.hpp>
+#include <gazebo_ros/conversions/gazebo_msgs.hpp>
```

Comment:
Did you mean to add a test here?

Reply:
I couldn't think of any test that may be required. Actually I saw `sensor_msgs` being included without a test so I thought I should include it.

Reply:
humm interesting, the sensors header is not needed. Ideally we'd have unit tests for all conversion functions. Right now they're being indirectly tested through `gazebo_plugins` tests.

Reply:
Should I remove this for now?

Reply:
yeah do you mind removing both sensors and gazebo? Thanks!